app/components/primer/clipboard_copy_component.ts

```diff
@@ -1,5 +1,7 @@
 import '@github/clipboard-copy-element'
 
+const CLIPBOARD_COPY_TIMER_DURATION = 2000
```

Comment:
We can just inline this variable since it's only used once.

Reply:
I only extracted it for clarity, since someone that is not very familiar with JS wouldn't have to guess what `2000` is in that function. But I can inline it if you really prefer :)